migrations/2_deploy_admin_contract.js

```diff
@@ -2,17 +2,26 @@ const AllowlistUtils = require('../scripts/allowlist_utils');
 
 const Admin = artifacts.require("./Admin.sol");
 
+async function logCurrentAdmins(instance) {
+    let currentAdmins = await instance.getAdmins();
+    console.log("\n<<< current ADMIN list >>>");
+    console.log(currentAdmins);
+}
```

Comment:
I like what you did for the allow list, where you logged the end of the list. 
console.log("\<<< end of current ADMIN list >>>");

Reply:
done